charts/gatekeeper/values.yaml

```diff
@@ -10,6 +10,7 @@ validatingWebhookFailurePolicy: Ignore
 validatingWebhookCheckIgnoreFailurePolicy: Fail
 enableDeleteOperations: false
 enableExternalData: false
+enableTLSHealthcheck: false
```

Comment:
`/charts` shouldn't be modified. When we cut a new release we copy over `manifest_staging/charts` to `/charts`. Can you make modifications in `manifest_staging/charts`? 

https://github.com/open-policy-agent/gatekeeper/tree/master/cmd/build/helmify/static#contributing-changes

Reply:
gotit